modules/apps/29-fee/keeper/escrow.go

```diff
@@ -214,21 +215,16 @@ func (k Keeper) RefundFeesOnChannelClosure(ctx sdk.Context, portID, channelID st
 				return err
 			}
 
-			// if the refund address is blocked, skip and continue distribution
-			if k.bankKeeper.BlockedAddr(refundAddr) {
-				continue
-			}
-
 			// refund all fees to refund address
-			// Use SendCoins rather than the module account send functions since refund address may be a user account or module address.
-			moduleAcc := k.GetFeeModuleAddress()
-			if err = k.bankKeeper.SendCoins(cacheCtx, moduleAcc, refundAddr, packetFee.Fee.Total()); err != nil {
-				return err
+			if err = k.bankKeeper.SendCoinsFromModuleToAccount(cacheCtx, types.ModuleName, refundAddr, packetFee.Fee.Total()); err != nil {
+				failedToSendCoins = true
+				continue
 			}
-
 		}
 
-		k.DeleteFeesInEscrow(cacheCtx, identifiedPacketFee.PacketId)
+		if !failedToSendCoins {
```

Comment:
it's possible an error occurred sending the coins, we don't want to remove them unless they were actually sent.